src/libraries/System.Private.CoreLib/src/System/ArgumentException.cs

```diff
@@ -97,5 +99,19 @@ private void SetMessageField()
         }
 
         public virtual string? ParamName => _paramName;
+
+        /// <summary>Throws an <see cref="ArgumentNullException"/> if <paramref name="argument"/> is null. Throws an <see cref="ArgumentException"/> if <paramref name="argument"/> is an empty string.</summary>
+        /// <param name="argument">The reference type argument to validate as non-null and non-empty.</param>
+        /// <param name="paramName">The name of the parameter with which <paramref name="argument"/> corresponds.</param>
```

Comment:
You can add `<Exception/>` nodes to describe the exact exceptions that will be thrown, if desired.